applications/PfemFluidDynamicsApplication/custom_elements/updated_lagrangian_element.cpp

```diff
@@ -1619,6 +1619,37 @@ namespace Kratos
     return std::sqrt(2.0 * NormS);
   }
 
+  template <>
+  void UpdatedLagrangianElement<2>::ComputeMechanicalDissipation(ElementalVariables &rElementalVariables)
+  {
+    KRATOS_TRY;
+    const double volumetric_strain = (rElementalVariables.SpatialDefRate[0] + rElementalVariables.SpatialDefRate[1]) * 0.5;
+
+    double mechanicalDissipation = rElementalVariables.UpdatedDeviatoricCauchyStress[0] * (rElementalVariables.SpatialDefRate[0] - volumetric_strain) +

```

Comment:
const

Reply:
usually we use mechanical_dissipation for this kind of local variables

Reply:
done!